cel/README.md

```diff
@@ -32,7 +32,7 @@ cel-controller-654bdc4cc8-7bvvn   1/1     Running   0          3m4s
 ## Usage
 
 To evaluate a CEL expressions using `Custom Tasks`, we need to define a [`Run`](https://github.com/tektoncd/pipeline/blob/master/docs/runs.md)
-type with `apiVersion: cel.tekton.dev/v1alpha1` and `kind: CEL`. The `Run` takes the CEL expressions to be evaluated
+type with `apiVersion: cel.dev/v1alpha1` and `kind: CEL`. The `Run` takes the CEL expressions to be evaluated
```

Comment:
I'm not sure we should squat domains we don't control. `cel.tekton.dev` sounded fine to me, or I've been using `example.dev` while it's experimental.

Reply:
Makes sense, fixed it to be `cel.tekton.dev` everywhere